packages/plattform-komponenter/src/person-card/icons/Child.tsx

```diff
@@ -1,24 +1,17 @@
 import React from 'react';
+import { FigureChildFillIcon } from '@navikt/aksel-icons';
 
-const ChildIcon = () => (
-  <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 39.8 39.8" xmlSpace="preserve">
-    <title>Barn</title>
-    <path
-      style={{ fillRule: 'evenodd', clipRule: 'evenodd', fill: '#78706A' }}
-      d="M19.9,0C8.9,0,0,8.9,0,19.9s8.9,19.9,19.9,19.9s19.9-8.9,19.9-19.9C39.7,8.9,30.8,0,19.9,0z"
-    />
-    <path
-      style={{ fillRule: 'evenodd', clipRule: 'evenodd', fill: '#FFFFFF' }}
-      // eslint-disable-next-line max-len
-      d="M12,21.9c0,0.6,0.5,1.1,1.1,1.1c0.4,0,0.8-0.2,1-0.5l1.9-3.1l-1.4,7.4h2.5v5.7c0,0.6,0.5,1.2,1.2,1.2 c0.6,0,1.2-0.5,1.2-1.2v-5.7l0,0V15.1H18c-1.6,0-1.9,0.2-2.5,1.1l-3.2,5C12.1,21.4,12,21.6,12,21.9z"
-    />
-    <path
-      style={{ fillRule: 'evenodd', clipRule: 'evenodd', fill: '#FFFFFF' }}
-      // eslint-disable-next-line max-len
-      d="M20.3,15.1v10.7l0,0v6.8c0,0.6,0.5,1.2,1.2,1.2c0.6,0,1.2-0.5,1.2-1.2v-6.8h1.1v-6.3l1.9,3.1 c0.2,0.3,0.5,0.5,0.9,0.5c0.6,0,1.1-0.5,1.1-1.1c0-0.3-0.1-0.5-0.3-0.7l-3.2-5c-0.6-0.9-1-1.1-2.6-1.1L20.3,15.1L20.3,15.1z"
-    />
-    <circle style={{ fill: '#FFFFFF' }} cx="19.9" cy="10.3" r="4.1" />
-  </svg>
+interface Props {
+  size?: number;
+}
+
+const ChildIcon = ({ size = 24 }: Props) => (
```

Comment:
Treng ein wrapparar for desse ikona? (Eg har ikkje sjekka bruken utover bruk i denne PR'en)